metricbeat/module/system/process/process.go

```diff
@@ -0,0 +1,114 @@
+/*
+
+An example event looks as following:
+
+
+    {
+      "@timestamp": "2016-04-26T19:24:19.108Z",
+      "beat": {
+        "hostname": "ruflin",
+        "name": "ruflin"
+      },
+      "metricset": "process",
+      "module": "system",
+      "rtt": 20982,
+      "system-process": {
```

Comment:
Mentioning @jpountz here as he can perhaps also shed some light on what the above would mean from the elasticsearch perspective?